stl/inc/xutility

```diff
@@ -796,18 +804,20 @@ concept weakly_incrementable = movable<_Ty> && requires(_Ty __i) {
     requires _Signed_integer_like<iter_difference_t<_Ty>>;
     { ++__i } -> same_as<_Ty&>;
     __i++;
-};
+}
+#ifdef __clang__ // TRANSITION, LLVM-48173
+&& !same_as<_Ty, bool>
+#endif // TRANSITION, LLVM-48173

```

Comment:
oddball formatting.

Reply:
Yeah, I hate tacking things on after a _requires-expression_ even without the added preprocessor conditional confusion.
I'll add a layer of indent here and see if it helps.